class-components/src/scripts/components/BottomSection.tsx

```diff
@@ -0,0 +1,131 @@
+import { Component } from 'react';
+import {
+  BottomSectionState,
+  SearchState,
+  Species,
+  Planets,
+  Vehicles,
+  Starships,
+  Films,
+  People,
+  Item,
+} from '../types/interfaces';
+
+class BottomSection extends Component<SearchState, BottomSectionState> {
+  constructor(props: SearchState) {
+    super(props);
+    this.state = {
+      items: [],
+      error: false,
+    };
+  }
+
+  componentDidMount() {
+    this.fetchItems(this.props.searchTerm);
+  }
+
+  componentDidUpdate(prevProps: SearchState) {
+    if (prevProps.searchTerm !== this.props.searchTerm) {
+      this.fetchItems(this.props.searchTerm);
+    }
+  }
+
+  fetchItems = async (searchTerm: string) => {
+    const endpoints = searchTerm
+      ? [
+          'https://swapi.dev/api/people/',
+          'https://swapi.dev/api/films/',
+          'https://swapi.dev/api/starships/',
+          'https://swapi.dev/api/vehicles/',
+          'https://swapi.dev/api/species/',
+          'https://swapi.dev/api/planets/',
+        ]
+      : ['https://swapi.dev/api/people/'];
```

Comment:
Please create a const for `https://swapi.dev/api/` in a separate file and use it across the project.
Please create an `enum` for the possible paths - e.g. `people`, `films` and so on, and use it across the project.